src/components/Tabs/TabPanel/TabPanel.jsx

```diff
@@ -4,12 +4,12 @@ import cx from "classnames";
 import useMergeRefs from "../../../hooks/useMergeRefs";
 import "./TabPanel.scss";
 
-const TabPanel = forwardRef(({ className, id, children }, ref) => {
+const TabPanel = forwardRef(({ className, id, children, index }, ref) => {
   const componentRef = useRef(null);
   const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
 
   return (
-    <div ref={mergedRef} className={cx("tab-panel--wrapper", className)} id={id} role="tabpanel">
+    <div key={id || index} ref={mergedRef} className={cx("tab-panel--wrapper", className)} id={id} role="tabpanel">
```

Comment:
this could break old tabs implementations if someone pass numeric id for only some of the tabs (few tabs will have the same numeric key and it can cause weird behivior) 
what about a combine  key with the index and the id?

Reply:
@hadasfa numeric id's are very bad practice and can break some browser api's - 

> Note: Technically, in HTML5, the value for an id attribute may contain any character, except whitespace characters. However, to avoid inadvertent errors, only ASCII letters, digits, '_', and '-' should be used and the value for an id attribute should start with a letter. For example, . has a special meaning in CSS (it acts as a class selector). Unless you are careful to escape it in the CSS, it won't be recognised as part of the value of an id attribute. It is easy to forget to do this, resulting in bugs in your code that could be hard to detect.

so i wouldn't worry about it.